model/model.wl

```diff
@@ -687,166 +663,197 @@ evaluateScenario[state_, fitParams_, standardErrors_, stateParams_, scenario_, n
 
   distancing = stateDistancingPrecompute[state][scenario["id"]]["distancingFunction"];
 
-  PCRQuantiles[t_] := simDeciles[#[PCR][t]&] * population;
-  DeathQuantiles[t_] := simDeciles[#[Deaq][t]&] * population;
-  DailyPCRQuantiles[t_] := simDeciles[#[PCR][t] - #[PCR][t-1]&] * population;
-  DailyDeathQuantiles[t_] := simDeciles[#[Deaq][t]-#[Deaq][t-1]&] * population;
+  CurrentlySuseptibleQuantiles[t_] :=  simDeciles[#[Sq][t]&] * population;
   CurrentlyInfectedQuantiles[t_] := simDeciles[#[Eq][t]&] * population;
   CurrentlyInfectiousQuantiles[t_] := simDeciles[#[Iq][t]&] * population;
-  CumulativeHospitalizedQuantiles[t_] := simDeciles[#[RepHq][t]*population &];
-  CumulativeCriticalQuantiles[t_] := simDeciles[#[RepHCq][t]*population&];
-  (* TODO why does this not include people who are exposed or infectious? *)
-  (* it looks like this used later on to mock up the number of people who are symptomatic; we cannot compute this quantity with the current model *)
-  CumulativeInfectionQuantiles[t_] := simDeciles[#[Deaq][t] + #[Rq][t]&] * population;
-  CumulativeRecoveredQuantiles[t_] := simDeciles[#[Rq][t]&] * population;
-  RecoveredHospitalizedQuantiles[t_] := simDeciles[#[RHq][t]&] * population;
-  RecoveredCriticalQuantiles[t_] := simDeciles[#[RCq][t]&] * population;
   CurrentlyHospitalizedQuantiles[t_]:=simDeciles[#[HHq][t]&] * population;
   CurrentlyCriticalQuantiles[t_]:=simDeciles[#[CCq][t]&] * population;
+  CumulativeRecoveredQuantiles[t_] := simDeciles[#[Rq][t]&] * population;
```

Comment:
I reorganized this section, renaming some of the functions for consistency, and removed a few that were no longer used in the codebase.